environments/azure.yaml

```diff
@@ -230,4 +230,4 @@ allure:
     enabled: true
     hosts:
       - paths: 
-          - /allure-docker-service*
+          - /allure-docker-service/*
```

Comment:
You need to add       
kubernetes.io/ingress.class: azure/application-gateway
      appgw.ingress.kubernetes.io/appgw-ssl-certificate: buying-catalogue-digital-nhs-uk 
      appgw.ingress.kubernetes.io/ssl-redirect: "true"